### ISSUE TITLE
Error: '.' is not recognized as an internal or external command (when running npm run add-project)

### ISSUE THREAD

**Issue by tomastrajan** [angular/ngcc-validation#372]
I have tried running following commands

`npm run add-project @angular-extensions/model`
`npm run add-project angular-extensions/model`
`npm run add-project angular-extensions-model`

to make sure it's not because of the special chars...

Could it be because of Window? 

EDIT:

```
Angular CLI: 9.0.0-next.4
Node: 10.11.0
OS: win32 x64
Angular: 9.0.0-next.6
... animations, common, compiler, compiler-cli, core, elements
... forms, language-service, platform-browser
... platform-browser-dynamic, platform-server, router
... service-worker, upgrade

Package                            Version
------------------------------------------------------------
@angular-devkit/architect          0.803.3
@angular-devkit/build-angular      0.900.0-next.4
@angular-devkit/build-optimizer    0.900.0-next.4
@angular-devkit/build-webpack      0.900.0-next.4
@angular-devkit/core               7.3.9
@angular-devkit/schematics         7.3.9
@angular/cdk                       8.2.0
@angular/cli                       9.0.0-next.4
@angular/fire                      5.2.1
@angular/flex-layout               8.0.0-beta.27
@angular/http                      7.2.15
@angular/localize                  9.0.0-next.7
@angular/material                  8.2.0
@angular/material-experimental     8.2.0
@angular/material-moment-adapter   8.2.0
@ngtools/webpack                   9.0.0-next.4
@nguniversal/aspnetcore-engine     8.1.1
@nguniversal/common                8.1.1
@nguniversal/express-engine        8.1.1
@schematics/angular                8.3.3
@schematics/update                 0.803.5
rxjs                               6.5.3
typescript                         3.5.3

```

**Comment by mgechev** [comment]
[Could be](https://github.com/angular/ngcc-validation/blob/master/infra/create-proj.ts#L9-L11) because of Windows. Not sure if anyone has tested it on Windows yet.

**Comment by gkalpak** [comment]
This should have been fixed by #433. If not, please open a new issue.
